Ports/SuperTuxKart/package.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=SuperTuxKart
+useconfigure='true'
+version='1.4'
+archive_hash='9890392419baf4715313f14d5ad60746f276eed36eb580636caf44e2532c0f03'
+files="https://github.com/supertuxkart/stk-code/releases/download/${version}/supertuxkart-${version}-src.tar.xz ${port}-${version}-src.tar.xz $archive_hash"
+auth_type='sha256'
+workdir="${port}-${version}-src"
+launcher_name='SuperTuxKart'
+launcher_category='Games'
+launcher_command='/usr/local/bin/supertuxkart'
+depends=(
+    'curl'
+    'freetype'
+    'glu'
+    'harfbuzz'
+    'libjpeg'
+    'libopenal'
+    'libogg'
+    'libpng'
+    'libvorbis'
+    'openssl'
+    'SDL2'
+    'sqlite'
+    'zlib'
+)
+
+configopts=(
+    '-DCMAKE_BUILD_TYPE=STKRelease'
+    "-DCMAKE_INSTALL_PREFIX=${SERENITY_INSTALL_ROOT}/opt/SuperTuxKart"
+    "-DSTK_INSTALL_DATA_DIR=${SERENITY_INSTALL_ROOT}/opt/SuperTuxKart"
+    '-DSUPERTUXKART_DATADIR=/opt/SuperTuxKart'
+    "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+    '-DBUILD_RECORDER=OFF'
+    '-DBUILD_SHARED_LIBS=OFF'
+    '-DUSE_WIIUSE=OFF'
+    '-DUSE_DNS_C=ON'
+    '-Bbuild'
+    '-GNinja'
+    "-DCMAKE_C_FLAGS=-I${SERENITY_INSTALL_ROOT}/usr/include/LibGL -I${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2 -I${SERENITY_INSTALL_ROOT}/usr/local/include -lcurl -lharfbuzz -lfreetype"
```

Comment:
It shouldn't be necessary to include the install root in these `-I`  flags, we are passing --sysroot to the compiler via the toolchain file.

I'm curious why we need to manually include and link these things. aren't there -DUSE_SDL or -DUSE_CURL or -DUSE_FREETYPE cmake options we can pass to make it use find_package to find the libs?

Reply:
the `-I` flags were needed probably due to a `/usr/local/include` vs `/usr/include` discrepancy. Both SDL, cURL and freetype are required for the package to build - for whatever reason the linker decided not to do that on Serenity, and we couldn't figure out why.

Reply:
Fair enough, but probably worth another look when updating the port next